.eslintrc.js

```diff
@@ -78,6 +78,16 @@ module.exports = {
         }],
         'strict': 0,
       },
+      'overrides': [
+        {
+          'files': [
+            'src/**/ion-string-handler.js',
+          ],
+          'rules': {
+            'complexity': [2, 20],
+          },
+        },
+      ]
```

Comment:
For single-file overrides, it is generally recommended to add a line to the top of the file.

Mind adding `/* eslint complexity: [2, 20] */` to the top of `ios-string-handler.js` instead?

Reply:
Sounds good. done!